modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util/cookie/cookie.js

```diff
@@ -19,6 +19,7 @@ const generateCookie = (name, value, options = {}) => {
 	for (const [key, value] of Object.entries(options)) {
 		if (key === 'secure') {
 			cookie += value ? '; secure' : '';
+			continue;
```

Comment:
@dsanz IIRC, Carol and you knew most about this. Was there some edge case in formats this code was trying to fix? If the idea is to output booleans in reduced format, `secure` is not the only one. I see `HttpOnly` in dev tools

![image](https://github.com/user-attachments/assets/53619bd9-ef1f-4509-8a5f-623ea34560b9)

Maybe we want here something like
```js
if (typeof value === 'boolean') {
    cookie += `; ${key};
}
else {
    cookie += `; ${key}=${value}`;
}
```
What do you think?

Reply:
Yep I was looking into this, and I'd say for flags we shall use the compact format. Both chrome and FF seem not to understand `secure=false` but rather expect flag not to be present.

About httponly, I doubt we can generate them in the browser direcly, but still I do support your suggestion @markocikos 

@istvansajtos please apply Marko's suggestion and check SF. Thank you